src/WinGet.RestSource.Utils/Validators/StringValidators/DisplayNameValidator.cs

```diff
@@ -1,28 +1,28 @@
-﻿// -----------------------------------------------------------------------
-// <copyright file="DisplayVersionValidator.cs" company="Microsoft Corporation">
-//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
-// </copyright>
-// -----------------------------------------------------------------------
-
-namespace Microsoft.WinGet.RestSource.Utils.Validators.StringValidators
-{
-    /// <summary>
-    /// DisplayVersionValidator.
-    /// </summary>
-    public class DisplayVersionValidator : ApiStringValidator
-    {
-        private const bool Nullable = true;
-        private const uint Min = 1;
-        private const uint Max = 128;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="DisplayVersionValidator"/> class.
-        /// </summary>
-        public DisplayVersionValidator()
-        {
-            this.AllowNull = Nullable;
-            this.MinLength = Min;
-            this.MaxLength = Max;
-        }
-    }
-}
+﻿// -----------------------------------------------------------------------
+// <copyright file="DisplayNameValidator.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.Utils.Validators.StringValidators
+{
+    /// <summary>
+    /// DisplayNameValidator.
+    /// </summary>
+    public class DisplayNameValidator : ApiStringValidator
+    {
+        private const bool Nullable = true;
+        private const uint Max = 100;
```

Comment:
>100 [](http://example.com/codeflow?start=33&length=3)

DisplayName is 256 from the spec. Is this for something else?

Reply:
No it's for that. There where just so many I made a few mistakes and missed some of the numbers.

Reply:
Thats what CRs are for.